Apps/Playground/0.64/windows/Playground.sln

```diff
@@ -39,11 +39,11 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.ReactNative.Manag
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.ReactNative.Managed.CodeGen", "..\node_modules\react-native-windows\Microsoft.ReactNative.Managed.CodeGen\Microsoft.ReactNative.Managed.CodeGen.csproj", "{ADED4FBE-887D-4271-AF24-F0823BCE7961}"
 EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "RNPermissions", "..\node_modules\react-native-permissions\windows\RNPermissions\RNPermissions.vcxproj", "{99677B9D-A27B-4239-930E-C36C8D339C54}"
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "BabylonReactNative", "..\..\..\..\Modules\@babylonjs\react-native-windows\windows\BabylonReactNative\BabylonReactNative.vcxproj", "{0362370A-ED67-41E5-85FC-394F27993115}"
 EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "SliderWindows", "..\node_modules\@react-native-community\slider\windows\SliderWindows\SliderWindows.vcxproj", "{685A83AE-36BC-4E9D-BDC6-417EBF168463}"
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "SliderWindows", "..\..\0.64\node_modules\@react-native-community\slider\windows\SliderWindows\SliderWindows.vcxproj", "{685A83AE-36BC-4E9D-BDC6-417EBF168463}"

```

Comment:
to my big surprise, yes. autolink will fail with something else. I didn't find a solution to that.

Reply:
Wow that's wild, I guess we can't comment it either since it is a solution file. Did you manually edit the .sln file for this? If you remove the project from the VS UI and then add it back, does the path stay the same?

Reply:
at first, I changed it manually but got autolink error (--check), so in a second step, I removed the lines and run a command to generate the autolink.